### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by guyd**
- added the documentation of the blocked-list
- clarified the usage section
- fixed several formatting and maintainability issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,87 +2,122 @@
...
+
+### Sample '.pre-commit-config.yaml'
+
+Here is an example of what your pre-commit-config.yaml may look like:
+
+``` yaml
 repos:
   - repo: https://github.com/privateai/pai-pre-commit-hook.git
-    rev: v1.2.1-beta
+    rev: v1.2-beta
```
Comment (koryf) [inline README.md:68]:
```suggestion
    rev: v1.2.1-beta
```

Updating the example to the latest version

File: README.md [hunk 2]
```diff
@@ -2,87 +2,122 @@
...
     hooks:
       - id: pii-check
         args:
           [
             "--url",
-            "URL",
+            "http://localhost:8080/v3/process/text",
```
Comment (koryf) [inline README.md:74]:
```suggestion
            "http://localhost:8080/v3/process_text",
```
Suggest keeping this as such until we publish beta3 with the new endpoint

**Review by koryf — APPROVED** [review]
LGTM with some comments!

File: README.md [hunk 3]
```diff
@@ -2,87 +2,122 @@
...
-            "ENV_FILE_PATH",
+            ".env",
             "--enabled-entities",
-            "ENTITY-1",
-            "ENTITY-2",
+            "PASSWORD",
+            "ORGANIZATION",
+            --blocked-list,
+            "MY_ID",
+            "ID-\d+"
```
Comment (ketakipai) [inline README.md:82]: Not sure what this is for?

File: README.md (hunk 1 above)
Comment (ketakipai) [reply README.md:68]: The latest version name is actually decided by us when we make a release, are you saying you prefer to name the release I'll do once this PR is in as '1.2.1-beta'?

**Review by ketakipai — APPROVED** [review]
LGTM. The way you've explained the disabling part with a use case made it a lot clearer. Thanks!

File: README.md (hunk 3 above)
Comment (guyd) [reply README.md:82]: yes

**Pull request merged**